src/box/rt_lua_call_access.c

```diff
@@ -0,0 +1,101 @@
+#include "rt_lua_call_access.h"
+#include "diag.h"
+#include "assoc.h"
+#include "user.h"
+#include "session.h"
+#include "../lua/init.h" /* tarantool_lua_is_builtin_global */
+
+static struct mh_access_t *user_rt_access_cache = NULL;
+static struct mh_strnptr_t *user_rt_universe_access_cache = NULL;
```

Comment:
These hash maps are not caches, they are the primary source of information about the runtime privileges. I think that the current `*_cache` name may be confusing.

Reply:
Removed postfix `*_cache`.